### ISSUE TITLE
[BasiliskII] Failed to run in MacOS 10.14.6

### ISSUE THREAD

**Issue by rickyzhang82** [cebix/macemu#187]
I pulled from master commit 01052ca89d22882c870b84c5498cd0ee9a843b6b and built in Mac OS 10.14.6 with the following build configuration.

```
make clean
NO_CONFIGURE=1 ./autogen.sh
./configure --enable-sdl-video --enable-sdl-audio --disable-vosf --disable-jit-compiler --with-x --with-gtk --with-mon
make -j 32
```

I could build BII. GTK configuration UI looks good. But when I hit start button, the screen is blank.

I can build and run fine in Fedora Linux `5.2.5-200.fc30.x86_64`. It seems it is recent Mac OS upgrade cause the break. 

Does anyone know what went wrong in Mac OS X?

**Comment by MBeijer** [comment]
I think it has to do with SDL1.2 not working properly on latest MacOS iterations.
Mojave doesn't support OpenGL, and they're removing support for 32bit binaries as well.

There has been work done to get BasiliskII to work on SDL2 #118

**Comment by emendelson** [comment]
Have you tried the kanjitalk755 fork? It runs perfectly well in Mojave and Catalina:

https://github.com/kanjitalk755/macemu

**Comment by rickyzhang82** [comment]
@emendelson 

I tried the repo in Linux. The Gtk configuration UI is broken. I had to disable to GUI. But it seems that it is working in SDL2.

But in Mac OS X, I got segmentation fault with the following built configuration:

```
./configure --enable-sdl-video --enable-sdl-audio --disable-vosf --disable-jit-compiler --with-mon
```

They might only make the Mac OS X build working in XCode. You know XCode keep on changing over the time. The only thing doesn't change that much is autoconf. It is portable in general.

If I had time and energy, I might consider integrate SDL2 to BII. But I'm happy with BII in Linux now.
